milhoja/cli.py

```diff
@@ -15,16 +24,32 @@
     help=u'Run as if milhoja was started in <path> instead of the current working directory.',
     type=click.Path()
 )
+@click.option(
+    u'--verbose',
+    default=False,
+    is_flag=True,
+    help=u'Enables the debug logging.'
+)
 @click.pass_context
-def main(ctx, c):
+def main(ctx, c, verbose):
     """Script entry point for Milhoja commands.
 
     Arguments:
     ctx -- CLI context.
     c -- Path where to run milhoja
+    verbose -- Enables debug logging
     """
     ctx.obj = dict()
-    ctx.obj['target'] = c
+    ctx.obj.update({
+        'target': c,
+        'verbose': verbose
+    })
+
+    if verbose:
```

Comment:
Nit:

```
level = logging.DEBUG if verbose else .logging.INFO
logger.setLevel(level)
```

Reply:
you can even do one liner `logger.setLevel(logging.DEBUG if verbose else logging.INFO)`